_maps/map_files/Delta/delta.dmm

```diff
@@ -71997,8 +71991,12 @@
 /turf/simulated/floor/plasteel,
 /area/station/maintenance/apmaint)
 "dTh" = (
-/obj/machinery/door/airlock/maintenance,
-/obj/effect/turf_decal/stripes/line,
+/obj/effect/turf_decal/delivery,
+/obj/machinery/camera{
+	c_tag = "Virology south";
```

Comment:
Is this that same departures camera? Looks like it might need to be re-labeled (and re-networked).

Reply:
fixed